docs/openapi/components/schemas/common/CertificateOfOriginCertificate.yml

```diff
@@ -0,0 +1,134 @@
+$linkedData:
+  term: CertificateOfOriginCertificate
+  '@id': https://w3id.org/traceability#CertificateOfOriginCertificate
+title: Certificate Of Origin
+description: Certifications made about the origin of a consignment.
+type: object
+properties:
+  '@context':
+    type: array
+    const:
+      - 'https://www.w3.org/2018/credentials/v1'
+      - 'https://w3id.org/traceability/v1'
+  type:
+    type: array
+    const:
+      - VerifiableCredential
+      - CertificateOfOriginCertificatez
+  id:
+    type: string
+  name:
+    type: string
+  description:
+    type: string
+  issuanceDate:
+    type: string
+  issuer:
+    type: object
+  credentialSubject:
+    type: object
+  proof:
+    type: object
+  relatedLink:
+    title: Related Link
+    description: Links related to this verifiable credential
+    type: array
+    items:
+      $ref: ./LinkRole.yml
+additionalProperties: false
+required: []
+example: |-
+  {
+    "@context": [
+      "https://www.w3.org/2018/credentials/v1",
+      "https://w3id.org/traceability/v1"
+    ],
+    "id": "urn:uuid:326c74c8-f4d7-4c1b-b158-2683deb8768e",
+    "type": [
+      "VerifiableCredential",
+      "CertificateOfOriginCertificate"
+    ],
+    "name": "Certificate of Origin",
+    "issuanceDate": "2019-12-11T03:50:55Z",
+    "issuer": {
+      "type": "Organization",
+      "id": "did:key:z6MktHQo3fRRohk44dsbE76CuiTpBmyMWq2VVjvV6aBSeE3U",
+      "name": "North Italy Chamber of Commerce"
+    },
+    "credentialSubject": {
+      "type": [
+        "Consignment"
+      ],
+      "shipper": {
+        "type": "Organization",
+        "name": "Espresso Italiano Co.",
+        "address": {
+          "type": "PostalAddress",
+          "streetAddress": "Via Vico Ferrovia 5",
+          "addressLocality": "Goro",
+          "addressRegion": "Ferrara",
+          "postalCode": "44020",
+          "addressCountry": "IT"
+        },
+        "email": "sales@espresso-italiano.example.com",
+        "phoneNumber": "+39 0351 9067195"
+      },
+      "consignee": {
+        "type": "Organization",
+        "name": "Prosumer Coffee Supplies, Ltd.",
+        "description": "Coffee Machine Imports",
+        "address": {
+          "type": "PostalAddress",
+          "streetAddress": "3934 Spinnaker Lane",
+          "addressLocality": "Joliet",
+          "addressRegion": "Illinois",
+          "postalCode": "60432",
+          "addressCountry": "US"
+        }
+      },
+      "carrier": {
+        "type": "Organization",
```

Comment:
This seems off, looks like you have an organization type that has manufacturing country and total number of packages on it?
im guessing this is a mistake in the generation of the example

Reply:
Ah yes! Definitely a mistake... Good catch.